docs/OMS-Agent-for-Linux.md

```diff
@@ -686,7 +690,7 @@ The default settings are
 ```
 
 # Uninstalling the OMS Agent for Linux
-The agent packages can be uninstalled using dpkg or rpm, or by running the bundle .sh file with the `--remove` argument. Additionally, if you want to completely remove all pieces of the OMS Agent for Linux you can run the bundle .sh file with the `--purge` arguement. 
+The agent packages can be uninstalled using dpkg or rpm, or by running the bundle .sh file with the `--remove` argument. Additionally, if you want to completely remove user data such as certificate and agent configurations of the OMS Agent for Linux you can run the bundle .sh file (or onboard_agent.sh file) with the `--purge` argument.
```

Comment:
Thanks, one more thing, can you rephrase "if you want to completely remove user data such as certificate and agent configurations of the OMS Agent for Linux you can run the bundle .sh file "

to be "if you want to remove user data such as certificate and agent configurations of the OMS Agent for Linux you need to run the bundle .sh file "

Reply:
Done